bob-access/src/authenticator/basic.rs

```diff
@@ -60,10 +60,14 @@ impl<Storage: UsersStorage> Basic<Storage> {
                 );
         
                 let user = self.users_storage.get_user(&username)?;
-                if user.password() == password {
+                if username == "default" {
```

Comment:
There is interesting thing that empty password in users.yaml would be converted not in "" String but in "~" String. So I can hardcode "~" password in extractor in case of missing header, but don't look nice

Reply:
Looks like it is a bug in serde_yaml: https://github.com/dtolnay/serde-yaml/issues/87.
I think, we can overcome the problem by making `password` `Optional<string>` in `ConfigUser` and converting `None` into an empty string (https://github.com/qoollo/bob/blob/master/bob-access/src/authenticator/users_storage/config.rs#L29)

Reply:
Probably, the problem can also be resolved by supplying default value `#[serde(default = "...")]`